prover/src/trace/trace_lde/default/mod.rs

```diff
@@ -63,10 +63,11 @@ where
         trace_info: &TraceInfo,
         main_trace: &ColMatrix<E::BaseField>,
         domain: &StarkDomain<E::BaseField>,
+        num_partitions: usize,
     ) -> (Self, TracePolyTable<E>) {
         // extend the main execution trace and build a commitment to the extended trace
         let (main_segment_lde, main_segment_vector_com, main_segment_polys) =
-            build_trace_commitment::<E, E::BaseField, H, V>(main_trace, domain);
+            build_trace_commitment::<E, E::BaseField, H, V>(main_trace, domain, num_partitions);
```

Comment:
I would probably make `num_partitions` a filed in `DefaultTraceLde` and then we won't need to pass it again to `set_aux_trace()` method.

Reply:
Done, but then had to re-add it back when implementing the solution with the minimal partition size